.drone.star

```diff
@@ -1210,6 +1209,7 @@ def e2eTests(ctx):
         steps = skipIfUnchanged(ctx, "e2e-tests") + \
                 restoreBuildArtifactCache(ctx, "yarn", ".yarn") + \
                 restoreBuildArtifactCache(ctx, "playwright", ".playwright") + \
+                restoreBuildArtifactCache(ctx, "web-dist", "dist") + \
                 installYarn() + \
```

Comment:
```suggestion
                installYarn() + \
                restoreBuildArtifactCache(ctx, "web-dist", "dist") + \
```
IMO this will be the flow for running steps.

Reply:
I won't matter though